### PR TITLE
Update TileMap doc descriptions for update_bitmask_area/region

### PR DESCRIPTION
**Pull request by CyanBlob**

### CODE REVIEW

File: doc/classes/TileMap.xml [hunk 1]
```diff
@@ -198,6 +199,8 @@
 			<argument index="1" name="end" type="Vector2" default="Vector2( 0, 0 )">
 			</argument>
 			<description>
+				Apply autotiling rules for the cells in given region (specified by grid-base X and Y coordinates)
+				Calling with invalid (or missing) paramaters will apply autotiling rules for the entire TileMap
```
Comment (Noshyaar) [inline doc/classes/TileMap.xml]: >paramaters

**Comment by CyanBlob** [comment]
Woops, sorry about that. I corrected the spelling and added the missing periods. Thanks!

File: doc/classes/TileMap.xml [hunk 2]
```diff
@@ -188,6 +188,7 @@
 			<argument index="0" name="arg0" type="Vector2">
 			</argument>
 			<description>
+				Apply autotiling rules for the cell referenced by its grid-based X and Y coordinates, and its surrounding cells.
```
Comment (cbscribe) [inline doc/classes/TileMap.xml]: - Use present tense in all method descriptions: Apply -> Applies
- Remove unnecessary comma

File: doc/classes/TileMap.xml [hunk 3]
```diff
@@ -198,6 +199,8 @@
 			<argument index="1" name="end" type="Vector2" default="Vector2( 0, 0 )">
 			</argument>
 			<description>
+				Apply autotiling rules for the cells in given region (specified by grid-base X and Y coordinates).
```
Comment (cbscribe) [inline doc/classes/TileMap.xml]: - Apply -> Applies
- for -> to
- in given region -> in the given region
- grid-base -> grid-based

File: doc/classes/TileMap.xml [hunk 4]
```diff
@@ -198,6 +199,8 @@
...
+				Calling with invalid (or missing) parameters will apply autotiling rules for the entire TileMap.
```
Comment (cbscribe) [inline doc/classes/TileMap.xml]: - will apply -> applies

**Review by cbscribe — CHANGES_REQUESTED** [review]
Style changes - please see http://docs.godotengine.org/en/latest/community/contributing/docs_writing_guidelines.html

**Pull request merged**